fast/stages/02-networking-nva/README.md

```diff
@@ -81,36 +81,53 @@ This is an options summary:
 
 ### IP ranges, subnetting, routing
 
-These are the subnets allocated by default in this setup:
+Minimizing the number of routes (and subnets) in the cloud environment is important, as it simplifies management and it avoids hitting [Cloud Router](https://cloud.google.com/network-connectivity/docs/router/quotas) and [VPC](https://cloud.google.com/vpc/docs/quota) quotas and limits. For this reason, we recommend to carefully plan the IP space used in your cloud environment. This allows the use of larger IP CIDR blocks in routes, whenever possible.
+
+This stage uses a dedicated /16 block (10.128.0.0/16), which should be sized to the own needs. The subnets created in each VPC derive from this range.
+
+The /16 block is evenly split in eight, smaller /19 blocks, assigned to different areas of the GCP network: *landing untrusted europe-west1*, *landing untrusted europe-west4*, *landing trusted europe-west1*, *landing untrusted europe-west4*, *development europe-west1*, *development europe-west4*, *production europe-west1*, *production europe-west4*.
+
+The first /24 range is carved from each /19 and effectively allocated for the subnet.
```

Comment:
`The first /24 range is carved from each /19 and effectively allocated for the subnet.`
->
`The first /24 on every area is allocated for a default subnet, which can be removed or modified as needed`